### ISSUE TITLE
Several mentions of ogcapi_common in OGC API - Coverages - Part 

### ISSUE THREAD

**Issue by ghobona** [opengeospatial/ogcapi-code-sprint-2021-07#19]
mentions of ogcapi_common should be changed to ogcapi-common

**Comment by jerstlouis** [comment]
@ghobona Are you talking about the links to opengis.net/spec ?
e.g. `http://www.opengis.net/spec/ogcapi_common-1/1.0/req/core`

I imagine the Coverages ones as well should be fixed, e.g.

`http://www.opengis.net/spec/ogcapi_coverage-1/` should be  `http://www.opengis.net/spec/ogcapi-coverage-1/` ?

**Comment by jerstlouis** [comment]
@ghobona 

Like this: https://github.com/opengeospatial/ogcapi-coverages/commit/2af589845cf3e060181cb2e4f60ae0597bafc354 ?

**Comment by ghobona** [comment]
@jerstlouis Yes, mentions of `ogcapi_coverage` in the URIs should be changed to `ogcapi-coverages`.

That is, in addition to replacing the underscore with a hyphen, the resource name should be plural (i.e. coverages).

**Comment by jerstlouis** [comment]
@ghobona Fixed and pushed https://github.com/opengeospatial/ogcapi-coverages/commit/5b700fdebc008e7048d9fcab8dbee4d69f9f9162
